iothub_client/tests/iothubtr_amqp_tel_msgr_ut/iothubtr_amqp_tel_msgr_ut.c

```diff
@@ -1665,11 +1665,109 @@ static TELEMETRY_MESSENGER_HANDLE create_and_start_messenger2(TELEMETRY_MESSENGE
     return handle;
 }
 
+char* umock_stringify_BINARY_DATA()
```

Comment:
The same applies here about what was mentioned for AMQP disposition above. These are not true copies. Have you run this code on linux under valgrind as well to check for memory leaks?

Reply:
same as above